### ISSUE TITLE
Allow completions on `docker compose`

### ISSUE THREAD

**Issue by rlnt** [greymd/docker-zsh-completion#5]
Would you be able to provide the completions for the more modern `docker compose` statement without a dash?

**Comment by pappasam** [comment]
Yeah, the README claims to support compose v2 (since that's where the link directs), but it does not appear to support it yet.

**Comment by greymd** [comment]
It should also provide completions for `docker compose` (without Dash) since this repository import the docker/cli completion file. However, this does not seem to work at this time due to the following issue.
Please wait for this issue to be resolved.
https://github.com/docker/cli/issues/3955

**Comment by sabahmax-dev** [comment]
I’ve opened PR #6 to address this by adding completion support for the modern `docker compose` syntax (without the dash).
